lib/portlayer/network/context.go

```diff
@@ -430,9 +459,13 @@ func (c *Context) BindContainer(h *exec.Handle) ([]*Endpoint, error) {
 	endpoints := con.Endpoints()
 	for _, e := range endpoints {
 		ne := h.ExecConfig.Networks[e.Scope().Name()]
-		ne.Static = &net.IPNet{
-			IP:   e.IP(),
-			Mask: e.Scope().Subnet().Mask,
+		ne.Static = nil
+		ip := e.IP()
```

Comment:
not for this PR, but we may be able to make consumption simpler by returning an IPNet that preserves the full IP address.